### PR TITLE
[GALL-2979] - Starting a placeholder for ViewingRoomsApp and route

### PR DESCRIPTION
**Pull request by oxaudo**
Thanks @damassi and @ansor4 for bearing with me here.

This is by no means a finished page but a beginning of `/viewing-rooms` page here.

@pvinis - I'm hoping in the next step here we can see how to create the VRs Grid reusing some of those cards I believe you've created for iOs view. Maybe we can pair on that at some point?

### CODE REVIEW

File: src/v2/Apps/ViewingRoom/ViewingRoomsApp.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    return null
+  }
+
+  const viewingRoomsForLatestGrid = viewingRooms.edges
+    .map(vr => {
+      if (!vr.node) {
+        return null
+      }
+
+      if (vr.node.status != "scheduled" && vr.node.status != "live") {
```
Comment (oxaudo) [inline src/v2/Apps/ViewingRoom/ViewingRoomsApp.tsx:24]: This filtering might become redundant if our MP will be able to request VRs by status.

File: src/v2/Apps/ViewingRoom/ViewingRoomsApp.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    })
+    .filter(vr => vr)
+
+  return (
+    <AppContainer maxWidth="100%">
+      <Box maxWidth={breakpoints.xl} mx="auto" width="100%">
+        <Sans size="10" my={3}>
+          Viewing Rooms
+        </Sans>
+        <Box>
```
Comment (oxaudo) [inline src/v2/Apps/ViewingRoom/ViewingRoomsApp.tsx:40]: Here I think we should create something like `ViewingRoomsGrid` that probably takes `viewingRoomsForLatestGrid` and just renders them properly. But was not sure how to break that up immediately.

File: src/v2/Apps/ViewingRoom/ViewingRoomsApp.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      }
+    })
+    .filter(vr => vr)
+
+  return (
+    <AppContainer maxWidth="100%">
+      <Box maxWidth={breakpoints.xl} mx="auto" width="100%">
+        <Sans size="10" my={3}>
+          Viewing Rooms
+        </Sans>
```
Comment (oxaudo) [inline src/v2/Apps/ViewingRoom/ViewingRoomsApp.tsx:39]: Featured rail will go here. Not clear if we'll need a separate MP call for FeaturedRooms or we can just add `featured` to the viewing rooms call here.

File: src/v2/Apps/ViewingRoom/ViewingRoomsApp.tsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+      if (vr.node.status != "scheduled" && vr.node.status != "live") {
+        return null
+      }
+
+      return {
+        ...vr.node,
+      }
+    })
+    .filter(vr => vr)
```
Comment (damassi) [inline src/v2/Apps/ViewingRoom/ViewingRoomsApp.tsx:32]: can also do `filter(Boolean)`

**Pull request merged**